### DISCUSSION TITLE
Practica 1 Abogabot

### DISCUSSION THREAD

**Discussion by Javicho9804** [Launch-X-Latam/MisionFrontEnd#225] [Show and tell]
Buenas noches gente, aqui poniendome al corriente con las practicas 😄 

Les comparto mi avance y espero contar con su retroalimentacion.

Cualquier cosa en lo que les pueda ayudar, aqui estoy para servirles✋ 

[Abogabot](https://github.com/Javicho9804/Abogabot)

**Comment by HoracioTorrez** [comment]
Se ve genial Javier. Yo estoy por terminar el mio también, oye podrías decirme como subir mis archivos al repositorio de github, muchas gracias y espero que no te moleste mi duda... Saludos

**Reply by Javicho9804** [reply]
@HoracioTorrez 
Claro brother, al editar el archivo README tan solo arrastra los archivos y automaticamente se suben 

Se te generaria algo asi :

![image](https://user-images.githubusercontent.com/114206571/196753363-c1abccab-cb3d-4336-8dd5-e95d8f84336a.png)


y en vista previa ya se estaria viendo asi :
![image](https://user-images.githubusercontent.com/114206571/196753613-40af3d2b-3d06-41f9-ad80-c19bb62c0e26.png)

Esa es la manera en lo que yo lo hice 👍 espero ayudarte

**Reply by HoracioTorrez** [reply]
Muchísimas gracias, me sirve de mucho tu ayuda... Saludos
